dactyl-config.yml

```diff
@@ -4897,9 +4897,11 @@ known_broken_links:
     - /blog/label/amendments.html
     - /blog/label/features.html
     - /blog/2023/nft-devnet-decommission.html
+    - carbon-calculator.html
```

Comment:
Please remove this line; it's addressed more properly by #1808 instead. Known broken links are for things that we can't fix the "right" way.